dd-sdk-android/src/main/kotlin/com/datadog/android/error/internal/DatadogExceptionHandler.kt

```diff
@@ -39,6 +42,10 @@ internal class DatadogExceptionHandler(
             e
         )
 
+        // give some time to the persistence executor service to finish its tasks
+        (CoreFeature.persistenceExecutorService as? ThreadPoolExecutor)
+            ?.waitToIdle(MAX_WAIT_FOR_IDLE_TIME_IN_MS)
```

Comment:
That's probably nit picking, but you don't use the return value at all here. 
So either you don't need to bother returning a boolean, or you might want to do something with the return value.

Reply:
Yes...I had to choose between returning a boolean or throwing a `TimeoutException`. I really don't want to throw an exception which in the end I will have to threat it somehow here. Instead returning a boolean is much more easier to handle, we could actually add an internal warning log in here notifying us that the persistence executor did not have time to finish. What do you think ? This way we can monitor how many times this happens.